core/src/main/java/org/apache/gravitino/listener/EventBus.java

```diff
@@ -47,6 +50,11 @@ public class EventBus {
    */
   public EventBus(List<EventListenerPlugin> eventListeners) {
     this.eventListeners = eventListeners;
+    this.asyncQueueListeners =
```

Comment:
What's the difference between the `eventListeners` and `asyncQueueListeners`? Can you explain more why do we need this and how they be used?

Reply:
`eventListeners` holds all event listeners include sync and async listeners while `asyncQueueListeners` includes async listeners only. The reason why we add `asyncQueueListeners` is to check the high watermark status of the async event queue to back pressure the linage sources.